release

```diff
@@ -0,0 +1,154 @@
+#!/bin/sh
+# vim: syntax=make ft=make
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+MAKEFILE_START_LINE=$(\
+  grep -ne '^# 8><' "$0" \
+  | tail -n 1 \
+  | awk -F: '{print $1}')
+
+LITANI_RELEASE=/tmp/litani-release
+mkdir -p "${LITANI_RELEASE}"
+
+MAKEFILE=$(mktemp ${LITANI_RELEASE}/XXXXXX || exit 1)
```

Comment:
I prefer `MAKEFILE=litani-release-makefile` only because it's clear to someone after running the release script what the file with the unfamiliar name actually is.